test/python/utils/test_deprecation.py

```diff
@@ -0,0 +1,157 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2022.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Tests for the methods in ``utils.deprecation``."""
+
+
+from qiskit import QiskitError
+from qiskit.utils.deprecation import deprecate_function, deprecate_arguments
+from qiskit.test import QiskitTestCase
+
+
+class DummyClass:
+    """This is short description. Let's make it
+    multiline"""
+
+    def __init__(self, arg1: int = None, arg2: [int] = None):
+        self.arg1 = arg1
+        self.arg2 = arg2
+
+    @deprecate_function(
+        "The DummyClass.foo() method is being deprecated. " "Use the DummyClass.some_othermethod()",
```

Comment:
lev'll eat you if he sees this re-formatted string concatenation :)

Reply:
nobody wants to be eaten by Lev. Fixed in 91c821d5e1345311fadea5e3502570246858cb99